Package.resolved

```diff
@@ -0,0 +1,43 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "GRDB",
+        "repositoryURL": "https://github.com/groue/GRDB.swift.git",
+        "state": {
+          "branch": null,
+          "revision": "2ac5c773537622b1520fdde3efc1a216119359ff",
+          "version": "5.24.1"
```

Comment:
Check whether different version in Podfile and here is cause discrepancy or not

Reply:
Yes I updated it to be of version 5.12.0 for GRDB, similar to pod